### PR TITLE
made some changes after doing code smell review. Please don't merge t…

### PR DESCRIPTION
**Pull request by kimasenbeck**
…hese changes until we have a chance to test this on a Neato! Feel free to comment though.

### CODE REVIEW

**Review by jovanduy — COMMENTED** [review]
Nice! Everything looks good. I like what you did with having the sign be a parameter in driveArc(). I'll approve once it's tested on a Neato.

File: auto_parking/scripts/parking.py [hunk 1]
```diff
@@ -60,8 +61,7 @@ def process_scan(self, m):
                 rospy.signal_shutdown("Done parking.")
             elif self.dist2Neato < 0.3:
                 print "Neato was too close to park!"
-            
-          
+
```
Comment (jovanduy) [inline auto_parking/scripts/parking.py:64]: I just realized: should we call rospy.signal_shutdown("Neato is too close to park.") right here?

Comment (kimasenbeck) [reply auto_parking/scripts/parking.py:64]: Yes! Maybe we should do the same if the spot is too small or if we never get a time 1 (maybe if too much time has passed and we haven't gotten one?)

File: auto_parking/scripts/parking.py [hunk 2]
```diff
@@ -74,52 +74,57 @@ def align_with_origin(self):
...
-            self.twist = FORWARD_SLOW
+            self.twist = FORWARD
         self.twist = STOP
         self.isAligned = True
 
-    def drive_arc(self, omega, travelTime, forward=False):
+        
+    def drive_arc(self, omega, travelTime, sign):
+        # The third parameter (sign) represents whether the forward velocity of the twist will be positive or negative
         if forward:
```
Comment (jovanduy) [inline auto_parking/scripts/parking.py]: this needs to be deleted!

**Review by jovanduy — APPROVED** [review]
I ran it on a Neato and got everything to work! I'll 🚢 it! (ship)

File: auto_parking/scripts/parking.py (hunk 1 above)
Comment (jovanduy) [reply auto_parking/scripts/parking.py:64]: We can make a separate pull request for this

**Pull request merged**